pkg/networkservice/loopback/server.go

```diff
@@ -53,9 +54,13 @@ func (l *loopbackServer) Request(ctx context.Context, request *networkservice.Ne
 	if err := createLoopback(ctx, l.vppConn, networkService, l.loopbacks, metadata.IsClient(l)); err != nil {
 		return nil, err
 	}
+	postponeCtxFunc := postpone.ContextWithValues(ctx)
+
```

Comment:
server side doesn't care about context expired on error in.

So added postponeCtxFunc.